docs/stylesheets/extra.css

```diff
@@ -0,0 +1,28 @@
+[data-md-color-scheme="aiod"] {
+  --md-primary-fg-color:        #0047BB; /* top header */
+  --md-primary-fg-color--light: #0047BB;
+  --md-primary-fg-color--dark:  #0047BB;
+
+
+  --md-typeset-a-color: #41B6E6;
+}
+
+/* AIOD colors
+dark blue: #0047BB
+alternative dark blue: #003399:
+light blue: #41B6E6
+yellow: #FFED00
+dark gray: #646567
+light gray: #C5C6C8
+ */
```

Comment:
Why are these properties commented?

Reply:
These are the specific AIoD color codes. I agree it's not great to keep them around here, but I didn't want to have to try look them up again later. At the same time, setting them as actual CSS variables is kind of meaningless since the theme won't use them with generic names. This seemed like a reasonable compromise in lieu of setting up something that's easier to reference in a separate repository.